src/material/autocomplete/autocomplete.html

```diff
@@ -3,6 +3,7 @@
     class="mat-mdc-autocomplete-panel mdc-menu-surface mdc-menu-surface--open"
     role="listbox"
     [id]="id"
+    [class]="_getPanelTheme()"
```

Comment:
I think that this and the `ngClass` on the next line might produce some weird results. We should combine them into a single binding.